cms/grading/ParameterTypes.py

```diff
@@ -94,6 +105,11 @@ class ParameterTypeString(ParameterType):
     TEMPLATE = "<input type=\"text\" name=\"{{parameter_name}}\" " \
         "value=\"{{parameter_value}}\" />"
 
+    def validate(self, value):
+        if not isinstance(value, string_types):
```

Comment:
Do you know of anything that would break if you used `str` here (meaning that they had to be `unicode` in py2)?